src/app/config.rs

```diff
@@ -540,8 +546,10 @@ impl Strings for Env {
 
 impl TestEnv {
     pub fn new() -> Self {
+        let mut values: HashMap<&'static str, String> = Default::default();
+        values.insert(ENV_PROXY_ID, "foo.deployment.default.linkerd-managed.linkerd.svc.cluster.local".into());
         Self {
-            values: Default::default(),
+            values: values,
```

Comment:
Nit: this can actually just be 
```suggestion
            values,
```
Rust has a nice shorthand for initializing a field from a name that's in scope.